sys/cheri/cheric.h

```diff
@@ -36,9 +36,10 @@
 #include <sys/cdefs.h>
 #include <sys/types.h>
 
-#if __has_feature(capabilities)
 #include <cheri/cherireg.h>	/* Permission definitions. */
 
+#if __has_feature(capabilities) || defined(__CHERI__)
```

Comment:
The __CHERI__ here seems to be a merge conflict gone awry?  I think that was removed in 'dev'.